db/re/item_db_etc.yml

```diff
@@ -33833,17 +33826,6 @@ Body:
     Script: |
       bonus bVariableCastrate,-15;
       /*Item removed on 2014-12-17*/
-  - Id: 23063
-    AegisName: Freeze_Dream
-    Name: Frozen Dream
-    Type: Etc
-    Trade:
-      NoDrop: true
-      NoTrade: true
-      NoCart: true
-      NoGuildStorage: true
-      NoMail: true
-      NoAuction: true
```

Comment:
Is this intentional?

Reply:
I guess the branch need an update!

Reply:
Was part of my instance database update. It's a consumable.
https://github.com/rathena/rathena/commit/c7ec076a6c1073f4ab9435c5b572cb8368698d16